core/src/main/scala/io/qbeast/core/transform/LinearTransformation.scala

```diff
@@ -185,6 +180,7 @@ class LinearTransformationDeserializer
       case (LongDataType, long: NumericNode) => long.asLong
       case (FloatDataType, float: DoubleNode) => float.floatValue
       case (DecimalDataType, decimal: DoubleNode) => decimal.asDouble
+      case (TimestampDataType, long: NumericNode) => long.asLong
```

Comment:
I will name the variable `timestamp` instead of `long` . I think it will make it more readable.